[2024-02-17T19:08:08.448+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: src_to_data_platform.src_to_platform_products_table manual__2024-02-17T19:07:52.114864+00:00 [queued]>
[2024-02-17T19:08:08.528+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: src_to_data_platform.src_to_platform_products_table manual__2024-02-17T19:07:52.114864+00:00 [queued]>
[2024-02-17T19:08:08.579+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-02-17T19:08:08.860+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): src_to_platform_products_table> on 2024-02-17 19:07:52.114864+00:00
[2024-02-17T19:08:08.932+0000] {standard_task_runner.py:60} INFO - Started process 210 to run task
[2024-02-17T19:08:08.965+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'src_to_data_platform', 'src_to_platform_products_table', 'manual__2024-02-17T19:07:52.114864+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/src_to_platform_dag.py', '--cfg-path', '/tmp/tmps5osnoty']
[2024-02-17T19:08:08.969+0000] {standard_task_runner.py:88} INFO - Job 59: Subtask src_to_platform_products_table
[2024-02-17T19:08:09.354+0000] {task_command.py:423} INFO - Running <TaskInstance: src_to_data_platform.src_to_platform_products_table manual__2024-02-17T19:07:52.114864+00:00 [running]> on host 8bc74b80747f
[2024-02-17T19:08:10.093+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='src_to_data_platform' AIRFLOW_CTX_TASK_ID='src_to_platform_products_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-17T19:07:52.114864+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-17T19:07:52.114864+00:00'
[2024-02-17T19:08:10.157+0000] {logging_mixin.py:188} INFO - Error: ('01000', "[01000] [unixODBC][Driver Manager]Can't open lib 'ODBC Driver 17 for SQL Server' : file not found (0) (SQLDriverConnect)")
[2024-02-17T19:08:10.184+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-17T19:08:10.259+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=src_to_data_platform, task_id=src_to_platform_products_table, execution_date=20240217T190752, start_date=20240217T190808, end_date=20240217T190810
[2024-02-17T19:08:10.460+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-17T19:08:10.580+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
